README.md

```diff
@@ -282,6 +282,7 @@ And finally, you are ready to go! Or at least you should be...
 **Iris Mestres** | [Contact me][deliaBlue]
 
 ### Junior Contributors
+**Ricard Garcia* |[Contact me][rigais23] | Llívia capital del món!
 
 
 
```

Comment:
For links to actually render in markdown, you need to add `[rigais23]` at the bottom following the format shown here!